crates/ruff_python_parser/src/lib.rs

```diff
@@ -305,6 +307,7 @@ pub struct Parsed<T> {
     syntax: T,
     tokens: Tokens,
     errors: Vec<ParseError>,
+    syntax_errors: Vec<SyntaxError>,
```

Comment:
We should rename this to `unsupported_syntax_errors` or similar. All errors in this struct are syntax errors.

Reply:
Yeah, I think we should rename this to something else, I like Micha's suggestion. I also want to point out that the `syntax_errors` method should be renamed as well otherwise it could be confused with `errors` method that currently exists on `Parsed`.